src/components/PostComment.jsx

```diff
@@ -0,0 +1,58 @@
+import {useEffect, useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {  postComment, getCommentsById } from '../utils/api';
+
+
+
+export default function PostComment() {
+    const [author, setAuthor] = useState("tickle122");
+    const [body , setBody] = useState("")
+    const {article_id} = useParams();
+    const [toggle, setToggle] = useState(false)
+    
+   
+
+    const handleBody = (event) => {
+      const value = event.target.value;
+      console.log(value)
+      setBody(value)
+     }
+  
+    const handleSubmit = (event) => {
+      event.preventDefault();
+
+      postComment(article_id, author, body).then((res) => { 
+          return res;
```

Comment:
you'll want to re-render the comments list so that you can see the new comment get added to the list instantly